examples/cicd/airflow.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+import joblib
+import pandas as pd
+from airflow.decorators import dag, task, short_circuit_task
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+
+
+@dag(dag_id='model_training_with_deepchecks_validation',
+     schedule_interval=None,
+     default_args={
+        'owner': 'airflow',
+        'start_date': datetime(2023, 1, 1),
+     },
+     params={
+         'bucket': 'deepchecks-public',
+         'data_key': 'matan/data.csv',
+         'train_path': 'train.csv',
+         'test_path': 'test.csv',
+         'model_path': 'model.joblib'
+     },
+     catchup=False)
+def model_training_dag():
+
+    @short_circuit_task
+    def validate_data(**context):
+        from deepchecks.tabular.suites import data_integrity
+        from deepchecks.tabular import Dataset
+
+        hook = S3Hook('aws_connection')
+        file_name = hook.download_file(key=context['params']['data_key'], bucket_name=context['params']['bucket'],
+                                       local_path='.')
+        data_df = pd.read_csv(file_name)
+        dataset = Dataset(data_df, label='label', cat_features=[])
+        suite_result = data_integrity().run(dataset)
+        suite_result.save_as_html('data_validation.html')
+        hook.load_file(
+            filename='data_validation.html',
+            key='results/data_validation.html',
+            bucket_name=context['params']['bucket'],
```

Comment:
Missing some comments in the code

Reply:
It's already very long, if someone will want to use that he'll understand the actions are basic